.github/codecov.yaml

```diff
@@ -38,26 +38,26 @@ component_management:
       paths:
         - api/v1beta2
     - component_id: common
-      name: pkg/common (u)
+      name: internal/common (u)
```

Comment:
does not exist

```
❯ ls internal/
authorino  controller  envoygateway  gatewayapi  istio  kuadrant  log  observability  openshift  policymachinery  ratelimit  reconcilers  utils  wasm
```

Reply:
@KevFan could i get your input on these changes. Looking back at the original copy of this file there was an entry for 
```
flag_management:
  individual_flags:
    - name: unit
      paths:
        - pkg/**
        - api/**
      carryforward: true
    - name: integration
      paths:
        - controllers/**
      carryforward: true
``` 
which doesn't seem to be present anymore. 

Lots of files have moved around now after the kubebuilder 4 migration so I want to be sure we're updating this file in the right way rather than simply updating the paths. 
Do we still want to maintain this configuration of unit test vs integration test coverage. I don't know the repo well enough to make an informed decision. 
cc @eguzki

Reply:
I would just have a coverage component per folder in  `internal`. The component helps when analyzing coverage results as  you can check coverage trends per component.

Reply:
I'd just 🔥 codecov... 
Who uses those metrics? This is _an honest_ question! It seems like whatever this thing says, we merge either way, so what do we use this for?

Reply:
While we discuss and take a decision on this matter, can you review https://github.com/Kuadrant/kuadrant-operator/pull/1277? (I am using codecov metrics when analyzing coverage to see if more tests are needed)